api_tests/lib/ledger_runner.ts

```diff
@@ -0,0 +1,93 @@
+import * as bitcoin from "./bitcoin";
+import * as toml from "toml";
+import * as fs from "fs";
+import { sleep } from "./util";
+import { execSync, spawn } from "child_process";
+
+const project_root: string = execSync("git rev-parse --show-toplevel", {
```

Comment:
❌ Pass in `working_dir` and `log_dir` into the `LedgerRunner` so you won't need to execute this line of code in each file. Liekwise, pass in `log_dir` into btsieve runner.